src/Microsoft.Health.Dicom.Core/Features/Store/StoreDatasetValidator.cs

```diff
@@ -138,6 +142,16 @@ string EnsureRequiredTagIsPresent(DicomTag dicomTag)
             catch (ElementValidationException ex)
             {
                 validationResultBuilder.Add(ex, queryTag);
+                const int value = 1;
+                _telemetryClient.GetMetric(
```

Comment:
It might behoove us to write new metrics as [`EventCounters`](https://learn.microsoft.com/en-us/dotnet/core/diagnostics/event-counters#consume-eventcounters) while we're considering the transition to OpenTelemetry. But we already have AppInsights metrics that need to be migrated, so perhaps one more isn't too bad lol